pkg/maintainer/spv/spv.go

```diff
@@ -1,22 +1,63 @@
 package spv
 
 import (
+	"bytes"
 	"context"
+	"encoding/hex"
+	"fmt"
 	"time"
 
 	"github.com/ipfs/go-log/v2"
 
 	"github.com/keep-network/keep-core/pkg/bitcoin"
+	"github.com/keep-network/keep-core/pkg/tbtc"
 )
 
 var logger = log.Logger("keep-maintainer-spv")
 
+const (
+	// Default value for history depth which is the number of blocks to look
+	// back from the current block when searching for past deposit sweep
+	// proposal submitted events. The value is the approximate number of
+	// Ethereum blocks in a week.
+	spvDefaultHistoryDepth = 40320
+
+	// Default value for the limit of transactions returned for a given wallet
+	// public key hash. The value is based on the frequency of how often deposit
+	// sweep and redemption transaction will happen. Deposit sweep transactions
+	// are assumed to happen every 48h. Redemption transactions are assumed to
+	// happen every 3h.
```

Comment:
We could also mention this small yet important wallet implementation detail.

One could argue we shouldn't mention wallet implementation details in the maintainer docs and there is a truth in it but... this is quite a crucial piece of knowledge and I think it's worth mentioning here in case we need to make some changes in the SPV maintainer logic in the future.

```suggestion
	// Default value for the limit of transactions returned for a given wallet
	// public key hash. The value is based on the frequency of how often deposit
	// sweep and redemption transaction will happen. Deposit sweep transactions
	// are assumed to happen every 48h. Redemption transactions are assumed to
	// happen every 3h. The wallet should refuse any proposals from the
	// coordinator if the previously executed Bitcoin transaction was not proved
	// to the Bridge yet so in theory, the value of 1 should be enough. We make
	// it a bit higher - better to be safe than sorry.
```

Reply:
Done in 72058add43edb65b5c2176770366dcc99ced7685.